web/index.html

```diff
@@ -29,6 +29,33 @@
 
 <body>
   <div id="app"></div>
+  <script>
+    if ('serviceWorker' in navigator) {
+      window.addEventListener('load', function () {
+        navigator.serviceWorker
+          .register('sw.js', { scope: '/' })
```

Comment:
You probably want to change this to match the same name as the old one so it'll overwrite the existing one.

https://github.com/nwhacks/nwhacks2018_static/blob/master/index.html#L26